Sources/Castor/Castor.docc/Articles/setup-and-lifecycle.md

```diff
@@ -0,0 +1,152 @@
+# Setup and Lifecycle
+
+@Metadata {
+    @PageColor(purple)
+}
+
+Integrate Google Cast into an iOS application using the ``Castor`` SDK.
+
+## Overview
+
+To integrate Google Cast into an iOS application with ``Castor``, you must first configure your project correctly.
+This includes obtaining the App ID for your receiver, adding the necessary network permissions in the _Info.plist_, and initializing the Cast context at launch.
+
+## Configure the project
+
+The App ID allows the application to identify and communicate with the designated receiver, while the network permissions ensure the app can discover and connect to compatible devices on the local network.
+
+### Get the App ID
+
+If you do not yet have an App ID, you can [create one](https://developers.google.com/cast/codelabs/cast-receiver#3) via the Google Cast Developer Console.  
+You can also use the default App ID provided by Google: `CC1AD845`.
+
+### Configure the _Info.plist_ file
+
+After obtaining your App ID, you must configure the _Info.plist_ file as follows:
+
+```xml
+<key>NSBonjourServices</key>
+<array>
+  <string>_googlecast._tcp</string>
+  <string>_<YOUR_APP_ID>._googlecast._tcp</string> <!-- Replace <YOUR_APP_ID> by CC1AD845 (default App ID) or your own App ID -->
+</array>
+
+<key>NSLocalNetworkUsageDescription</key>
+<string>${PRODUCT_NAME} uses the local network to discover Cast-enabled devices on your WiFi network.</string>
+```
+
+These entries are required for iOS to allow local network access and device discovery using _Bonjour_.
+
+### Add the dependency using SPM
+
+You can add ``Castor`` to your iOS project using SPM directly in Xcode.  
+Simply provide the ``Castor`` GitHub repository [URL](https://github.com/SRGSSR/castor).
```

Comment:
This is already mentioned in the main readme file, including more links to Xcode documentation and instructions for integration as dependency of another package.

I would therefore suggest either:

1. Remove this section entirely.
2. Or move it to the top and reuse the [instructions](https://github.com/SRGSSR/castor?tab=readme-ov-file#integration) from the main readme.